### PR TITLE
Air API Call and Storing into file

### PR DESCRIPTION
**Pull request by JrGkOG**
Created an api call and stored the json data.

From the inputs given to yash 
  - the api key is in a env file 
  - the file is named with uuid and not sha

### CODE REVIEW

**Comment by JrGkOG** [comment]
@raamsri sir have created the api call and done the storing using uuid 
took about 1 hour 25 mins for both of them 

was facing little bit of difficulty in both the json part and the api key part, so i was mostly reading docs to understand more , I felt that took a lot of time.

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module github.com/adofm/Maverick-Gliders.git
```
Comment (raamsri) [inline go.mod]: This must reference the upstream repo, which is `YashKarthik2927`; not your fork.

Comment (JrGkOG) [reply go.mod]: ok sir

**Review by raamsri — CHANGES_REQUESTED** [review]
Looks good to me @adofm . Just the module reference should be fixed. There are a few things that could be reworked as best practice but there's no rush; like, using a logger rather than fmt.Print and such. We'll incorporate them as the service grows. 

Good job!

**Comment by JrGkOG** [comment]
Ok sir will incorporate the changes

**Comment by JrGkOG** [comment]
Have changed the module reference to YashKarthik .

**Review by yendelevium — APPROVED** [review]

**Pull request merged**